content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -40,79 +40,19 @@ replaces the functionality provided by the `istio-init` container.
     *  The Kubernetes documentation highly recommends this for all Kubernetes installations
        where `ServiceAccounts` are utilized.
 
-1.  [Download the Istio release](/docs/setup/kubernetes/#downloading-the-release).
-
 ## Installation
 
 1.  Determine the Kubernetes environment's CNI plugin `--cni-bin-dir` and `--cni-conf-dir` settings.
+    Refer to [Hosted Kubernetes settings](#hosted-kubernetes-settings) for any non-default settings required.
 
-    1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
-        settings required.
-
-1.  Create a namespace for the Istio control-plane components:
-
-    {{< text bash >}}
-    $ kubectl create namespace istio-system
-    {{< /text >}}
-
-1.  Expand the release's compressed Istio CNI chart.
-
-    {{< text bash >}}
-    $ tar -xzf install/kubernetes/helm/charts/istio-cni-*.tgz -C install/kubernetes/helm/
-    {{< /text >}}
-
-1.  Install Istio CNI.
-
-    *  Option 1.  Render and apply the Istio CNI Helm template.
-
-        {{< text bash >}}
-        $ helm template install/kubernetes/helm/istio-cni --name=istio-cni --namespace=istio-system | kubectl apply -f -
-        {{< /text >}}
-
-    *  Option 2.  Install via Helm and Tiller via `helm install`.
-
-        {{< text bash >}}
-        $ helm install install/kubernetes/helm/istio-cni --name=istio-cni --namespace=istio-system
-        {{< /text >}}
+1.  Install Istio CNI and Istio using Helm.
+    Refer to the [Customizable Install with Helm](/docs/setup/kubernetes/install/helm/#cni) instructions and the **default with CNI enabled** profile.
```

Comment:
This link didn't seem to work for me maybe this is expected with things in transit.

Reply:
@john-a-joyce  What do you mean it doesn't work? The `#cni` hash at the end highlights the tabs that install the Istio CNI. Doesn't that work for you?